components/Candidates/CandidateSideBar.tsx

```diff
@@ -11,27 +12,30 @@ const experienceRadio = [
     { label: "5-6 years" },
     { label: "7-8 years" },
 ]
-const jobRadio =[
-    { label:"Full-time"},
+const jobRadio = [
+    { label: "Full-time" },
     { label: "Part-time" },
     { label: "Contract-based" },
     { label: "Freelance" },
     { label: "Internship" },
 ]
 export default function CandidateSideBar(): JSX.Element {
+    const [openJobCat, setOpenJobCat] = useState<boolean>(false);
+    const [openCountry, setOpenCountry] = useState<boolean>(false);
+    const [openSkills, setOpenSkills] = useState<boolean>(false);
     return (
         <div className='bg-[#F8F8F8] rounded-lg p-4 '>
-            <div className='mx-12'>
+            <div className='mx-0 md:mx-12'>
                 <div className='py-4'>
                     <h3 className='font-medium'>Sort by</h3>
 
                     <RadioBtn label={'Relevance'} value={'Relevance'} />
                     <RadioBtn label={'Date'} value={'Date'} />
                 </div>
                 <hr className='mt-6 mx-4 mb-12' />
-                <div>
+                <div className='w-full'>
                     <h3 className='font-medium mb-4'>Job Category</h3>
-                    <DropDown />
+                    <DropDown text="select category" options={["Health", "Technology", "Business"]} isOpen={openJobCat} onClick={() => setOpenJobCat(!openJobCat)} />
```

Comment:
I think the options array here will be better if it is declared outside

say const  optionsArray = [array values]

then options={optionsArray}